example/snippets.cpp

```diff
@@ -0,0 +1,805 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CppAlliance/url
+//
+
+//[snippet_headers_1
+#include <boost/url.hpp>
+//]
+
+#if 0
+//[snippet_headers_2
+#include <boost/url/src.hpp>
```

Comment:
Well, come on now - if you are going to put `#if 0` around these statements, then they aren't compiled. If they aren't compiled, then why bother to put them in the .cpp file? You might as well just put the statement in a C++ code block in the qbk file :)

Reply:
Yes. We can't get the compilation test in this case. My rationale was that it would much easier to remember to update snippet_headers_1 and snippet_headers_2 together in case anything needs to change.